<ui version="4.0" >
 <author>José Manuel Díez Botella</author>
 <class>consmayor</class>
 <widget class="QDialog" name="consmayor" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Consultas de Mayor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" >
      <item>
       <widget class="QLabel" name="textLabel1" >
        <property name="maximumSize" >
         <size>
          <width>110</width>
          <height>32767</height>
         </size>
        </property>
        <property name="font" >
         <font>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="frameShape" >
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Plain</enum>
        </property>
        <property name="text" >
         <string>Subcuenta:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="subcuentalineEdit" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maxLength" >
         <number>40</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buscapushButton" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Maximum" hsizetype="Maximum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset resource="keme.qrc" >
          <normaloff>:/images/librok-55.png</normaloff>:/images/librok-55.png</iconset>
        </property>
        <property name="shortcut" >
         <string/>
        </property>
        <property name="autoDefault" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>subcuentalineEdit</zorder>
     <zorder>buscapushButton</zorder>
     <zorder>textLabel1</zorder>
    </widget>
   </item>
   <item rowspan="2" row="0" column="1" >
    <widget class="QGroupBox" name="intervalogroupBox" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize" >
      <size>
       <width>170</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title" >
      <string>Intervalo cuentas</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="checked" >
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" >
        <item>
         <widget class="QLabel" name="label" >
          <property name="font" >
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text" >
           <string>Inicial:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="iniciallineEdit" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize" >
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" >
        <item>
         <widget class="QLabel" name="label_2" >
          <property name="font" >
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text" >
           <string>Final:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="finallineEdit" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize" >
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5" >
        <item>
         <widget class="QLabel" name="label_3" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font" >
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text" >
           <string>Intervalo:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="intervaloComboBox" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item rowspan="2" row="0" column="2" >
    <widget class="interFechas" native="1" name="fechas" />
   </item>
   <item rowspan="2" row="0" column="3" >
    <widget class="QGroupBox" name="groupBox_6" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" >
      <property name="spacing" >
       <number>0</number>
      </property>
      <property name="margin" >
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="copiarpushButton" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize" >
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="toolTip" >
         <string>Pasar al portapapeles el resultado de la consulta</string>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset resource="keme.qrc" >
          <normaloff>:/graf/Copy.png</normaloff>:/graf/Copy.png</iconset>
        </property>
        <property name="iconSize" >
         <size>
          <width>31</width>
          <height>31</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="editarpushButton" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="maximumSize" >
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="toolTip" >
         <string>Edición del asiento correspondiente</string>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset resource="keme.qrc" >
          <normaloff>:/images/contgen2.png</normaloff>:/images/contgen2.png</iconset>
        </property>
        <property name="iconSize" >
         <size>
          <width>31</width>
          <height>31</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="imprimirpushButton" >
        <property name="maximumSize" >
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="toolTip" >
         <string>Imprimir</string>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset resource="keme.qrc" >
          <normaloff>:/graf/Print.png</normaloff>:/graf/Print.png</iconset>
        </property>
        <property name="iconSize" >
         <size>
          <width>31</width>
          <height>31</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelarpushButton" >
        <property name="maximumSize" >
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="toolTip" >
         <string>Salir</string>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset resource="keme.qrc" >
          <normaloff>:/images/librok-52.png</normaloff>:/images/librok-52.png</iconset>
        </property>
        <property name="iconSize" >
         <size>
          <width>31</width>
          <height>31</height>
         </size>
        </property>
        <property name="shortcut" >
         <string/>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>cancelarpushButton</zorder>
     <zorder>imprimirpushButton</zorder>
     <zorder>editarpushButton</zorder>
     <zorder>copiarpushButton</zorder>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2" >
      <item>
       <widget class="QPushButton" name="refrescarpushButton" >
        <property name="maximumSize" >
         <size>
          <width>60</width>
          <height>50</height>
         </size>
        </property>
        <property name="toolTip" >
         <string>Refrescar</string>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset resource="keme.qrc" >
          <normaloff>:/images/librok-54.png</normaloff>:/images/librok-54.png</iconset>
        </property>
        <property name="iconSize" >
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="autoDefault" >
         <bool>false</bool>
        </property>
        <property name="default" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="previapushButton" >
        <property name="maximumSize" >
         <size>
          <width>60</width>
          <height>50</height>
         </size>
        </property>
        <property name="toolTip" >
         <string>Subcuenta anterior</string>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset resource="keme.qrc" >
          <normaloff>:/graf/1leftarrow.png</normaloff>:/graf/1leftarrow.png</iconset>
        </property>
        <property name="iconSize" >
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="siguientepushButton" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize" >
         <size>
          <width>60</width>
          <height>50</height>
         </size>
        </property>
        <property name="toolTip" >
         <string>Subcuenta posterior</string>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset resource="keme.qrc" >
          <normaloff>:/graf/1rightarrow.png</normaloff>:/graf/1rightarrow.png</iconset>
        </property>
        <property name="iconSize" >
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="4" >
    <widget class="QLabel" name="nombrelabel" >
     <property name="font" >
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>nombre empresa</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4" >
    <layout class="QHBoxLayout" name="HTableLayout" >
     <item>
      <widget class="QTableWidget" name="mayortable" />
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="4" >
    <widget class="QGroupBox" name="groupBox_4" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Maximum" hsizetype="Minimum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Acumulados extracto</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="spacing" >
       <number>6</number>
      </property>
      <property name="margin" >
       <number>9</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="layou_acu" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="margin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="textLabel1_2" >
          <property name="maximumSize" >
           <size>
            <width>80</width>
            <height>32767</height>
           </size>
          </property>
          <property name="text" >
           <string>DEBE:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="debelineEdit" >
          <property name="alignment" >
           <set>Qt::AlignRight</set>
          </property>
          <property name="readOnly" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textLabel2_2" >
          <property name="maximumSize" >
           <size>
            <width>80</width>
            <height>32767</height>
           </size>
          </property>
          <property name="text" >
           <string>HABER:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="haberlineEdit" >
          <property name="alignment" >
           <set>Qt::AlignRight</set>
          </property>
          <property name="readOnly" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textLabel3_2" >
          <property name="maximumSize" >
           <size>
            <width>80</width>
            <height>32767</height>
           </size>
          </property>
          <property name="text" >
           <string>SALDO:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="saldolineEdit" >
          <property name="alignment" >
           <set>Qt::AlignRight</set>
          </property>
          <property name="readOnly" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>interFechas</class>
   <extends>QWidget</extends>
   <header>src/interfechas.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>subcuentalineEdit</tabstop>
  <tabstop>buscapushButton</tabstop>
  <tabstop>refrescarpushButton</tabstop>
  <tabstop>previapushButton</tabstop>
  <tabstop>siguientepushButton</tabstop>
  <tabstop>cancelarpushButton</tabstop>
  <tabstop>imprimirpushButton</tabstop>
  <tabstop>editarpushButton</tabstop>
  <tabstop>debelineEdit</tabstop>
  <tabstop>haberlineEdit</tabstop>
  <tabstop>saldolineEdit</tabstop>
 </tabstops>
 <resources>
  <include location="keme.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>cancelarpushButton</sender>
   <signal>clicked()</signal>
   <receiver>consmayor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
